config.json

```
@@ -180,6 +180,14 @@
         "prerequisites": [],
         "difficulty": 2
       },
+      {
+        "slug": "secret-handshake",
+        "name": "Secret Handshake",
+        "uuid": "033c70bb-25cc-47ba-b4b7-92f0d012a5c8",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

```suggestion
        "difficulty": 4
```
This doesn't seem like a "2" exercise, it's not as easy as e.g. `leap`. Will set it to 4